### PR TITLE
Fix issues related to MultiResolutionFeatures

### LINKED ISSUE #1929: FDA: multiresolution feature extraction is not documented properly
the english is bad and the method is not explained clearly

### PR DESCRIPTION
**Pull request by pfistfl**
Fixes #1929

### CODE REVIEW

File: R/extractFDAFeaturesMethods.R [hunk 1]
```diff
@@ -177,20 +177,27 @@ extractFDAFPCA = function(pve = 0.99, npc = NULL) {
...
 #' @param curve.lens [\code{integer}]\cr
-#'   Curve subsequence lengths. Needs to sum up to the length of the functional.
+#'   Instead of splitting the curves in half from the top, a vector of sub-curves can be
+#'   can be specified. Specifies the curve subsequence's lengths.
+#'   Needs to sum up to the length of the functional.
 #' @return [\code{data.frame}].
 #' @export
 #' @family fda_featextractor
 extractFDAMultiResFeatures = function(res.level = 3L, shift = 0.5, curve.lens = NULL) {
+  assertInteger(res.level, len = 1L)
```
Comment (berndbischl) [inline R/extractFDAFeaturesMethods.R]: use res.level = asCount(res.level)

File: R/extractFDAFeaturesMethods.R [hunk 2]
```diff
@@ -177,20 +177,27 @@ extractFDAFPCA = function(pve = 0.99, npc = NULL) {
...
+  assertNumeric(shift, lower = 0L, upper = 1L, len = 1)
```
Comment (berndbischl) [inline R/extractFDAFeaturesMethods.R]: again, there are special asserts for scalars in cm. please read the vignette to know those

File: R/extractFDAFeaturesMethods.R [hunk 3]
```diff
@@ -177,20 +177,27 @@ extractFDAFPCA = function(pve = 0.99, npc = NULL) {
...
 #'
 #' @description
-#' The function extracts currently the mean of multiple segments of each curve and stacks them
-#' as features. The segments length are set in a hierachy way so the features
-#' cover different resolution levels.
+#' The function extracts the mean of multiple segments of each curve and extracts them
+#' as features. The segments length are set in a hierachy, so the features
+#' cover different resolution levels. This is done by sequentially dividing the
+#' functional up into smaller sub-curves of length l/2 and computing the mean of the
+#' sub-curves for a sliding window.
```
Comment (berndbischl) [inline R/extractFDAFeaturesMethods.R]: can we be really precise here? how does the shifting work?

**Comment by pfistfl** [comment]
Merged into #1937, closing here

**Pull request closed without merging**